src/evaluation/evaluation-error-handler.service.ts

```diff
@@ -124,7 +124,9 @@ export class EvaluationErrorHandlerService {
 
       // Email subject
       const processCode = currentEvaluationQueue?.processCode || 'N/A';
-      const emailSubject = `${processCode} Evaluation Feedback for ORIS code ${emailTemplateContext.orisCode} Unit ${emailTemplateContext.configuration}`;
+      const env = this.configService.get<string>('app.env')?.trim()?.toLowerCase();
+      const subjectSuffix = env && !['prod', 'production', ''].includes(env) ? ` (sent from ECMPS 2.0 ${env})` : '';
+      const emailSubject = `Error - ${processCode} Evaluation Feedback for ORIS code ${emailTemplateContext.orisCode} Unit ${emailTemplateContext.configuration} ${subjectSuffix}`;
```

Comment:
This will not include the "(sent from ECMPS 2.0)" portion in production. Is that the intent, or do we only want to leave off the environment?
```suggestion
      const emailSubject = `Error - ${processCode} Evaluation Feedback for ORIS code ${emailTemplateContext.orisCode} Unit ${emailTemplateContext.configuration} (sent from ECMPS 2.0${env && !['prod', 'production', ''].includes(env) ? ` ${env}` : ''})`;
```